test/jdk/java/awt/Focus/UnaccessibleChoice/AccessibleChoiceTest.java

```diff
@@ -132,6 +142,15 @@ public void start () {
 
         // On success second item should be selected
         if (choice.getSelectedItem() != choice.getItem(1)) {
+            // Print out os name to check if mac conditional is relevant
+            System.err.println("Failed on os: " + osName);
+
+            // Save image to better debug the status of test when failing
+            Rectangle screenRect = new Rectangle(100, 200,
+                    win.getWidth(), win.getHeight() + 20);
```

Comment:
Suggestion: You can capture the frame as well as window by using the following (it captures the entire testing screen).
```
GraphicsConfiguration ge = GraphicsEnvironment.getLocalGraphicsEnvironment().getDefaultScreenDevice().getDefaultConfiguration();
BufferedImage failImage = robot.createScreenCapture(ge.getBounds());
```

Reply:
I can update it to capture the screen instead if that's preferred. I initially had it this way, but I figured it was better to just show the window that is being tested alone. If it's better to capture the whole screen for debugging purposes, I can make this change.